packaging/suse/bin/portusctl

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+bundle="/srv/Portus/vendor/bundle/ruby/2.1.0/bin/bundler.ruby2.1"
+export GEM_PATH=/srv/Portus/vendor/bundle/ruby/2.1.0/
```

Comment:
please update https://github.com/SUSE/Portus/blob/master/packaging/suse/Portus.spec.in#L104